samples/AnomalyDetector/src/Generated/Docs/Multivariate.xml

```diff
@@ -33,7 +33,7 @@ Multivariate client = new AnomalyDetectorClient(endpoint, credential).GetMultiva
 
 Response response = await client.GetMultivariateBatchDetectionResultAsync(Guid.Parse("663884e6-b117-11ea-b3de-0242ac130004"), null);
 
-JsonElement result = JsonDocument.Parse(response.ContentStream).RootElement;
+JsonElement result = JsonDocument.Parse(response.ContentStream, new JsonDocumentOptions { MaxDepth = 256 }).RootElement;
```

Comment:
I think we do not really to update all the samples - samples are just showing how we could parse the result and get those properties, we do not have to show every detail

Reply:
Let's discuss this with a wider group to decide what should the sample behave.

Reply:
removed